platform-includes/performance/add-spans-example/javascript.mdx

```diff
@@ -3,56 +3,47 @@ For example, if you want to create a transaction for a user interaction on your
 ```javascript
 // Let's say this function is invoked when a user clicks on the checkout button of your shop
 function shopCheckout() {
-  // This will create a new Transaction for you
-  const transaction = Sentry.startTransaction({ name: "shopCheckout" });
-  // Set transaction on scope to associate with errors and get included span instrumentation
-  // If there's currently an unfinished transaction, it may be dropped
-  Sentry.getCurrentScope().setSpan(transaction);
-
-  // Assume this function makes a fetch call
-  const result = validateShoppingCartOnServer();
-
-  const span = transaction.startChild({
-    data: {
-      result,
-    },
-    op: "task",
-    description: `processing shopping cart result`,
+  return Sentry.startSpan({ name: "shopCheckout" }, () => {
```

Comment:
Here you can actually see how much nicer the new APIs are in such scenarios ❤️

Reply:
Yeah much cleaner, I agree.

One comment but logaf-l: Should we make this function `async` and `await` the fetch call? Might help users understand that `startSpan` supports async/Promise as well.

Reply:
good point, I think I'll just add both of these!